conf/salt/project/django/run_with_db.sh

```diff
@@ -0,0 +1,5 @@
+# Shell script to set up necessary environment variables and run a PostgreSQL command
+export PGHOST="{{ pillar['env']['DB_HOST'] }}"
+export PGUSER="{{ pillar['project_name'] }}_{{ pillar['environment'] }}"
+export PGPASSWORD="{{ pillar['secrets']['DB_PASSWORD'] }}"
+exec "$@"
```

Comment:
I added the conditional PGPORT setting, though with more Jinja than shell script for the logic.

Re: PGDATABASE; it is cute/handy/etc., but for me I like the command wrapped with the script to have the same form as it would without the wrapper when running the command in your dev environment.  I haven't made that change for now.